test/unit-tests/middleware/middlewareTest.js

```diff
@@ -288,7 +288,7 @@ describe('Test for middleware', () => {
         })
     })
 
-    it('Cna is undefined', function (done) {
+    it('Cna is undefined', (done) => {
```

Comment:
Right now they are used interchangeably since the only orgs supported in production so far are CNAs. However, when we start supporting other roles in the service the wording will have to change to refer to orgs specifically. I have made the change here for the purpose of this comment. However, continuous improvement in naming conventions should be addressed in a separate issue since this will require changes in different parts of the application.